_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #131"
+date: 2020-09-21 12:00:00 +0300
+---
+
+
```

Comment:
Since we want to support 2020.3 EAPs and drop 2020.1 support, let's add the corresponding note here.
See `123` release as an example